requirements.txt

```diff
@@ -0,0 +1,8 @@
+django==1.10
```

Comment:
What about Django 2?

Reply:
This is for master as it is, which is running on Django 1.10 in production right now.